client/analytics/report/products/table-variations.js

```diff
@@ -13,13 +13,17 @@ import { Link } from '@woocommerce/components';
 import { formatCurrency, getCurrencyFormatDecimal } from '@woocommerce/currency';
 import { getNewPath, getPersistedQuery } from '@woocommerce/navigation';
 import { numberFormat } from '@woocommerce/number';
+import { getSetting } from '@woocommerce/wc-admin-settings';
 
 /**
  * Internal dependencies
  */
 import ReportTable from 'analytics/components/report-table';
 import { isLowStock } from './utils';
 
+const manageStock = getSetting( 'manageStock', 'no' );
```

Comment:
Should we change some of these to booleans, if the only possible values are yes/no?

Reply:
Yup I think they should be.  Punt to another iteration?  I think we'd want to review all the settings and use better types.

Reply:
In the case of these core settings it might be best to do it 'now', mainly because a) if left it might be difficult to change this from string to bool without breakage, and b) we can see in the diff where these values are being used since those lines are being updated as part of this PR anyway.

Reply:
Hmm, I still would prefer to do in it's own pull because I'd want to review all instances of things that might be better typed.  It's not just that this pull is already larger but it also extends the time before its merged.

I _do_ intend on immediately releasing pulls that address things like this after this is merged.

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2943